1-org/envs/shared/log_sinks.tf

```diff
@@ -15,8 +15,10 @@
  */
 
 locals {
-  parent_resource_id   = local.parent_folder != "" ? local.parent_folder : local.org_id
-  parent_resource_type = local.parent_folder != "" ? "folder" : "organization"
+  #parent_resource_id   = local.parent_folder != "" ? local.parent_folder : local.org_id
+  parent_resource_id   = local.parent_folder != "" ? local.parent_folder : local.billing_account != "" ? local.billing_account : local.org_id
```

Comment:
I'm not sure if I got the intention of this expression right...

You're saying that terraform should use:
- `local.parent_folder` if provided;
- if not, check for `local.billing_account` and use it if provided;
- if not, use `local.org_id`.

But as far as I understood from your reports in our standups, shouldn't the billing account logs sinks **in parallel** of organization logs, and not **instead of** organization logs?